tests/test_workload.py

```diff
@@ -98,11 +96,11 @@ def test_column_eq(self):
 
         # Column name equal but table (for both) is None
         with self.assertRaises(Exception):
```

Comment:
Does `self.assertRaises(AssertionError):` work?

Reply:
Yes, I have exchanged these and all other appropriate occurrences with `AssertionError`.